openqa-schedule-mm-ping-test

```diff
@@ -10,6 +10,7 @@ flavor_override=${flavor_override:-mm-monitoring} # override flavor to avoid obs
 arch=${arch:-x86_64}
 version=${version:-Tumbleweed}
 test_name=${test_name:-ping_client}
+build_regex=${build_regex:-^\\d+$}
```

Comment:
Maybe it would be better to use `[0-9]` here, so we don't run into problems with escaped backslashes:
```suggestion
build_regex=${build_regex:-[0-9]+$}
```

Reply:
I assume we won't use double quotes in a regex for build names, though :)